ts/components/ContextualHelpModal.tsx

```diff
@@ -167,11 +168,17 @@ const ContextualHelpModal: React.FunctionComponent<Props> = (props: Props) => {
    * @param options Contains the checkboxes' values, @todo handle screenshot attachment request.
    */
   const handleContinue = (options: SupportRequestOptions) => {
+    const { supportToken, openBugReport } = props;
+
+    // See comment at line 232 to learn why this is not
```

Comment:
```suggestion
    // See comment below to know why this is not
```
If another developer add some lines in the file we should update also the line number